### DISCUSSION TITLE
Welche genaue Frequenz für HMS-Serie (CMT2300A) einstellen?

### DISCUSSION THREAD

**Discussion by huste511** [tbnobody/OpenDTU#1300] [Q&A]
Welche genaue Frequenz ist den in den Einstellungen zu wählen?
Oder kommt es technisch nicht auf +/- 5 Mhz an und der MWR paßt sich an und sendet auf der gleichen Frequenz zurück?

Hoymiles gibt ja nur "Sub-1G" an. ;-(
Sucht man etwas weiter, wird in https://www.hoymiles.com/?smd_process_download=1&download_id=13488 von einem 868-*Band* gesprochen. Das Band ist aber breit.

Im Datenblatt zu einer DTU findet man die EN 301489 welche die Emissionen von Funkgeräten in der EU im ISM-Band von 863 MHz bis 870 MHz regelt. Telemetrie, Datenkommunikation ist von 863-870 zugelassen.

Die Google AI sagt mir: "Die Hoymiles HMS-Serie verwendet das 868-MHz-Band für die Kommunikation zwischen den Wechselrichtern und dem Gateway. Die Geräte sind auf die Verwendung des Subbands von 868,0 MHz bis 868,6 MHz mit einer Bandbreite von 600 kHz optimiert. Die maximale Sendeleistung der Geräte beträgt 10 mW EIRP.
Frage ich nach der Quelle, rudert die AI zurück und sagt es wäre nicht genau definiert. ;-)

**Comment by tbnobody** [comment]
Wenn es funktioniert belasse einfach die Standardeinstellungen. Initial wird IMMER auf 868MHZ kommuniziert. Mit dem ersten Paket muss die DTU die Frequenz des Inverters einstellen. Das ist die Frequenz die in den Einstellungen dargestellt ist (865). Muss den Bandplänen usw. entsprechen. 
Wie gesagt. normal gibt es keinen Grund hier etwas zu ändern.

**Comment by huste511** [comment]
Ok, also ist 865 Mhz ein realer Wert der bei den meisten läuft. Dachte es wäre ein Standardwert, wo man ggf noch mal dran muß.
Habe mit 10mW und nur ca. 5m durchs Gebäude, immer wieder mal keine Datenübertragung für 20-40 Sek und bin noch etwas am suchen. Zu 99% läuft es, aber eben nicht komplett fehlerfrei.
Aktuell verwende ich eine Antenne aus Koax, die auf 8,6cm entmantelt wurde, für ein 1/4 Lamda-Dipol.

**Comment by turrican944** [comment]
In der Originalen DTU kannst du aus 16 verschiedenen Kanälen meine ich aussuchen. Ich hatte mal geschaut welcher Kanal welche Frequenz ist, aber irgendwie ist die Notiz grade weg. Es war jedenfalls nicht einfach Fortlaufend. 
Ich kann sonst bei Gelegenheit die Originale DTU nochmal mit dem RTL SDR abhorchen und mal eine Liste DTU Kanal zu Frequenz machen.
Meine läuft auch auf 865MHz war so eingestellt, ging und weit genug von 868.3MHz wo mein Homematic und noch diverse andere Sachen laufen weg

**Comment by huste511** [comment]
So eine Frequenzliste wäre super, besonders wenn diese im Netz zu finden ist.
DTU und SDR hat nicht jeder. Ich hab nur letzteres. ;-)
Dein SDR hat geringe Abweichung?
Hast Du schon mal geschaut ob die MWR dann genau auf dem gleichen Kanal oder sogar auf der gleichen Frequenz zurück senden?

**Comment by DerArzt94** [comment]
Konntet ihr eine Lösung finden? Habe das selbe Problem. Meine Sendeleistung steht mittlerweile auf 20dBm obwohl der Wechselrichter nur 1 Mauer und 10m entfernt ist. Trotzdessen gibt es ab und zu noch Probleme.

**Comment by SnakeGER** [comment]
Hallo.
Falls dieses Problem noch bei einigen aktuell ist, ich hatte bis gestern auch massive Probleme.

Kurz zu meiner Geschichte:
Ich habe seit Oktober ´23 zwei Wechselrichter (HMS-1600-4T)am laufen und habe mir dazu die AHOY-DTU gebaut (großes Lob an die Erfinder und Entwickler). Sporadisch hatte ich an einigen Tagen sehr schlechten Empfang zwischen Wechselrichter und AHOY. Die Fehlerrate (keine Antwort) war jenseits von 60%. Ich habe alles versucht, vom Neustart der DTU und Wechselrichter, über Up- und Downgrade der AHOY-Version, bis zum Trennen des Stroms, aber an diesem Tag konnte ich die Qualität der Verbindung nicht verbessern. Am nächsten Tag war meistens wieder alles OK (+-15% Verlustrate). Dies hielt dann wieder ein paar Tage an, um dann wieder einen ganzen Tag schlecht zu laufen. Natürlich habe ich versucht, die Frequenz zu ändern, aber die DTU baute keine Verbindung zum WR auf, worauf ich dann wieder die alte Frequenz eingestellt hatte, damit ich wenigstens etwas Empfang wiederherstellen konnte.
Die Lösung kam gestern, als ich mal wieder sehr schlechten Empfang hatte:

Ich habe in der DTU nochmals die Frequenzen geändert. Jeder WR hat jetzt seine eigene Frequenz. Vorher waren alle bei 863 MHz und jetzt bei 866 und 868 MHz. Man muss allerdings nach Änderung der Frequenz 10 Min warten und dabei die AHOY-DTU einfach laufen lassen (kann auch 15 Min gewesen sein). Die Wechselrichter übernehmen dann die Frequenz und senden auf dieser weiter.
Ich hatte schlagartig 0% Verlustrate und dementsprechend besten Empfang. Heute habe ich mal geguckt und ein WR ist bei 0,41% Verlust und der andere bei knapp 4% was aber noch völlig im grünen Bereich ist.
Ich kann nur vermuten, wieso es bei 863 MHz so schlecht lief. Vielleicht hat irgendein Nachbar auch ne DTU oder ne Alarmanlage oder nen Rasenmäher oder was weiß ich, welcher auch auf derselben Frequenz funkt und hat damit die Störungen in meiner DTU verursacht. Jedenfalls läuft es derzeit sehr gut.

Ich hoffe, meine Lösung kann noch weiteren Leuten bei Verbindungsproblemen helfen. Vielleicht sollte das Thema "Frequenzumstellung" mal in der AHOY Anleitung unter "Troubleshooting" mit aufgenommen werden. Bei Google findet man nämlich garnichts. Auch nicht, dass man nach Umstellung 10-15 Min warten muss, sonst hätte ich das schon viel eher versucht.

So, sorry für den langen Text, aber ich bin einfach happy, dass es bei mir geklappt hat und möchte somit anderen Leuten helfen.
Schreibt mal bitte, ob das bei euch geholfen hat.

Ach, ja: Abstand DTU-WR ca. 25m mit einer Mauer. RSSI: WR1 -63 dBm, WR2 -55 dBm. Intervall: 3 sec.

LG

**Comment by schwatter** [comment]
@SnakeGER 

Habe noch nie Ahoy probiert. So ganz habe ich es noch nicht verstanden, wie du die unterschiedlichen Frequenzen eingestellt hast. 
Bei OpenDtu wird eine Frequenz eingestellt, welche dann für alle gesetzt wird. Oder hast du immer einen WR am Strom, setzt die Frequenz und sobald er online ist, trennst du ihn vom Strom und nimmt dann denn anderen? 

Gruß schwatter

**Comment by mkroeber** [comment]
@tbnobody @huste511 @turrican944 
Woher weiß ich nun die genaue Frequenz?

Mein "Problem": ich habe eine dtu-lite-s laufen (brauchte ich wegen Netzbetreibereinstellungen) und zusätzlich ein openDTU.
Die openDTU hatte vor dem Einsatz der normalen orginalen DTU schon Verbindungsprobleme siehe hier #2200 .
Nun habe ich eine neue Antenne besorgt weil das offenbar ein Problem ist (#1879) und weiss nicht warum es immer noch nicht funktioniert...

Entweder habe ich an der openDTU die falsche frequenz eingestellt, oder meine Antenne funktioniert doch nicht so wie gewollt.
In der hoymiles-cloud finde ich keine Möglichkeit die Frequenz oder den Kanal auf dem gefunkt wird einzustellen. Da aber eine originale DTU verbunden ist, vermute ich, dass eine feste Frequenz von dieser verwendet wird und die open DTU nichts auf den "default" Kanälen findet,
Die Antenne die ich gekauft habe ist diese hier: https://www.amazon.de/dp/B07S24V6WQ?ref=ppx_yo2ov_dt_b_fed_asin_title&th=1

Ich werde heute noch mal probieren  ein 8,6cm (Lamda/4) langes Kabel als Antenne anzuschließen... meine "Angst" ist nur, dass ich trotz passender Antenne immer noch auf dem falschen Kanal unterwegs bin?

Vorab Danke für Eure Hilfe.
Gruß Markus

**Reply by turrican944** [reply]
Moin
Die originale DTU hat meine ich 8 oder 16 Kanäle, die kannst du mit der olle App einstellen. Welche Frequenz welcher Kanal ist habe ich nicht im Kopf. Die Frequenzen der einzelnen Kanäle waren irgendwie auch wild durch einander gewürfelt. Wie du nun die Frequenz raus bekommst ist einfach eine RLT SDR Stick nehmen und mit z.B. SDR Sharp schauen wo die originale DTU beim eingestellen Kanal sendet. Das kannst du dann für alle Kanäle machen und hast so zu jedem Kanal eine Frequenz. Ich meine Kanal 8 ware 866.750 MHz (aber ohne gewähr). Ich habe das auch nicht für alle Kanäle ausprobiert, wollte ich mal machen, habe die Lust daran aber verloren.
Wenn du nun einen Frequenz zum Kanal der Originalen DTU hast kannst du diese im Opendtu einstellen, aber auch damit funktioniert das nicht zuverlässig mit zwei DTUs parallel.  Das habe ich ausprobiert, es geht zuverlässig nur mit einer DTU.
Wenn du allerdings in beiden DTUs die gleiche Frequenz hast kannst du eine abschalten und die andere einschalten und bekommst sofort eine Verbindung.

Und warum überhaut die Originale die Netzeinstellungen kannst du damit doch einmal einstellen und gut, habe ich so auch gemacht, OpenDTU aus, die Originale eingeschaltet (meine hat den gleichen Kanal (habe die App aber nicht mehr auf dem Handy) verbindet die sich sofort (sonst musst du halt 15min warten) und dann stelle ich die Netzprofil sachen ein und tu das original ding wieder in die Ecke.
Das habe ich genau einmal gemacht, deutsches Netzprofil geändert (Q/U Kennline aktiviert) und eingespielt, seit dem liegt die original wieder in der Ecke.

Der Falsche Kanal ist egal so lange nur eine DTU läuft sollte nach spätesten 15Min alles wieder funktionieren, wenn du ungeduldig bist hilf auf Wechselrichter von den Solarpanelen zu trennen damit der komplett neustartet, schadet auch nicht, ich hatte das schon ein paar mal wenn ich die DTU zu oft tausche das es nicht mehr geht bis ich den WR Stromlos mache.

Das Setzen der Frequenz macht die DTU, der WR lauscht auf 868Mhz (genaue Frequenz ist erstmal auch egal) ob es dort eine DTU hört die ihm sagt gehe auf einen anderen Kanal. Das macht der WR so lange er keine Verbindung zur DTU hat, ist die Verbindung weg macht der WR das horchen auf 868Mhz wieder nach irgendwie 10-15min.
Aber nochmal OpenDTU und die original gleichzeitig haut nicht hin, evtl kann man testen ob das geht wenn man OpenDTU die gleiche Seriennummer gibt wie der Originalen, weil bei mir laufen zwei Opendtu problemlos an einem WR.

**Comment by SebStaeubert** [comment]
Ich betreibe 3 HMS-2000 an einer OpenDTU. An einigen Tagen kommt es zu Verbindungsschwierigkeiten (<10% erfolgreiche Anfragen) bei einzelnen WR, die auch durch einen Neustart der OpenDTU nicht zu beheben sind.
Kann es sein, dass sich die WR gelegentlich selbst stören und es sinnvoll ist, wie oben beschrieben, unterschiedliche Frequenzen je WR konfigurierbar zu machen?

**Comment by tah78** [comment]
Hier dasselbe Problem 3x HMT-2250-6t und openDTU. Alle WR sind mittlerweile max. 5m entfernt. RSSIs bei -44, -49 und -60 dBm. Der WR mit -60 dBm läuft manchmal ein paar Tage lang einwandfrei und dann geht es irgendwann wieder los, dass er nur morgens bei Sonnenaufgang und abendes bei Sonnenuntergang Daten sendet bzw. die openDTU diese empfängt.

Habe schon eine andere Stab-Antenne versucht, verschiedene Positionen der openDTU. Aber noch näher an alle 3 WR geht nicht. Ich überlege schon alle 18 PV-Module in einen String zur packen, mit Tigos auszustatten und an den großen WR anzubinden.

Hat schon einer mehr Erfolg, das Ganze stabil zum Laufen zu bekommen, insb. mit mehrere WR?

**Comment by turrican944** [comment]
Moin
Das werden wahrscheinlich Störungen sein, das irgendwas anderes im 868mhz band sendet uns stört.
Gelegentlich kommt das bei mir auch Mal vor das die WR für ein paar Stunden nicht erreichbar sind. War bislang aber immer wenn ich nicht vor Ort war sonst hätte ich schonmal geschaut ob zur der Zeit irgendwas anders sendet.
Das sich die WR gegenseitig stören glaube ich nicht. Die WR senden nur dann wenn die dtu sie anpingt.

Der rssi ist doch top bei mir läuft das stabil mit -72db

**Comment by SebStaeubert** [comment]
Bei mir ist es tagesweise durchwachsen. An manchen Tagen klappt es gut, heute sieht es so aus (Abbrüche bei HMS2):

Wechselrichter | dBm | erfolgreich % | teilweise % | Fehler: nichts empfangen %
-- | -- | -- | -- | --
HMS1 | -80 | 83 | 9 | 8
HMS2 | -81 | 17 | 43 | 40
HMS3 | -66 | 92 | 0 | 8

Manchmal hilft ein Frequenzwechsel, manchmal nicht.
Entfernung OpenDTU zu den Wechselrichtern ca. 15m. Es ist ein Baum innerhalb der Sichtlinie und die WR sind unter bzw. hinter den Modulen.

**Comment by tah78** [comment]
Hab auch mal die Frequenz gewechselt auf 868Mhz (vorher 865,25Mhz) geändert und die Sendeleistung auf -5dBm geändert. Hab gelesen, dass man damit auch rumspielen soll, wenn die WR sehr nah oder weiter weg sind. Ob das Auswirkungen hat und wie man den Erfolg von Anpassungen an der Sendeleistung erkennt/misst weiß ich aber nicht. Am RSSI sehe ich keine Änderungen, egal ob -5 oder +5 dBm. Aber wenn ihr stabile Verbindung mit -72 oder tlw. -80 habt, kann das ja auch nicht der Grund sein.

Heute läuft es. Empfang erfolgreich heute bei allen 3 WRs zwischen 73 und 75%. Alle 3 haben viele "Gesendete Fragment Wiederanforderungen". Bei 4000 Anfragen gesamt ca. 800 Wiederanforderungen. K.A. ob das viel ist, Bauchgefühl sagt ja, da ca. 20%. Aber was kann man dagegen tun? Abfrageintervall ändern? Meine werden aktuell alle 9 Sekunden abgefragt, d.h. das Intervall steht bei 3 Sekunden. Ich habe auch das Gefühl, dass das Ändern der Frequenz hilft. Aber wer oder was hier nach einer Weile schief läuft habe ich auch noch nicht herausgefunden. Da ich die WRs und Antenne der DTU nicht verändert habe, kann es ja nur noch der WR sein, der immer wieder Probleme macht oder die openDTU selbst?! In den Logs kommen viele Einträge. Ich kann diese aber leider weder einem WR zuordnen bzw. interpretieren, ob das nun besonders schlimm ist oder Hinweise auf die Ursache gibt:

`[14:04:32.239] W (367737541) hoymiles: Middle missing
[14:04:35.252] W (367740542) hoymiles: All missing
[14:04:38.240] W (367740543) hoymiles: Nothing received, resend whole request
[14:04:38.240] W (367743542) hoymiles: Middle missing
[14:04:41.242] W (367746545) hoymiles: Middle missing
[14:04:47.364] W (367752666) hoymiles: Middle missing
[14:04:50.364] W (367755666) hoymiles: Middle missing
[14:04:53.367] W (367758669) hoymiles: Middle missing
[14:04:56.371] W (367761671) hoymiles: Middle missing
[14:05:02.505] W (367767806) hoymiles: Middle missing
[14:05:05.599] W (367770805) hoymiles: Middle missing
[14:05:08.505] W (367773808) hoymiles: Middle missing
[14:05:11.510] W (367776813) hoymiles: Middle missing
[14:05:23.665] W (367788968) hoymiles: Middle missing
[14:05:26.676] W (367791970) hoymiles: Middle missing
[14:05:32.881] W (367798183) hoymiles: Middle missing
[14:05:35.881] W (367801183) hoymiles: Middle missing
[14:05:38.958] W (367804185) hoymiles: Middle missing
[14:05:41.886] W (367807188) hoymiles: Middle missing
[14:05:44.888] W (367810190) hoymiles: Last missing
[14:05:47.693] W (367813024) hoymiles: Frame kaputt
[14:05:48.054] W (367813035) hoymiles: Frame kaputt
[14:05:48.054] W (367813357) hoymiles: Middle missing
[14:05:53.696] W (367819027) hoymiles: Frame kaputt
[14:05:54.055] W (367819037) hoymiles: Frame kaputt
[14:05:54.055] W (367819358) hoymiles: Middle missing`

Was mir noch aufgefallen ist, das der RSSI bei dem betroffenen WR tagsüber stark schwankt, obwohl nichts in der Nähe verändert wurde. Es korreliert nicht mal mir der Leistung / Sonneneinstrahlung oder Temperatur, was ich noch vermutet hatte. Siehe hier, von heute:
<img width="1136" height="563" alt="Bildschirmfoto 2026-02-23 um 14 32 08" src="https://github.com/user-attachments/assets/337850a1-7117-4b79-9f22-f0b722a79e10" />
<img width="1120" height="428" alt="Bildschirmfoto 2026-02-23 um 14 33 22" src="https://github.com/user-attachments/assets/59aa3c03-18e0-4a32-ac66-e8b1ea6a6cf0" />
<img width="1127" height="412" alt="Bildschirmfoto 2026-02-23 um 14 33 32" src="https://github.com/user-attachments/assets/8db0e161-56bd-4da8-a683-446403aa39af" />

**Comment by turrican944** [comment]
So sieht es bei mir aus
<img width="1612" height="368" alt="image" src="https://github.com/user-attachments/assets/6d49b8b9-1ee7-4d3b-beaa-f6be3603f7f7" />
<img width="1565" height="456" alt="image" src="https://github.com/user-attachments/assets/2009aa2f-4f51-465a-b547-e50caf17588c" />
bei meinen beiden. Da sind auch fehlende Paket kommt bei meist aber morgens und abends vor wenn das Licht wohl nicht richtig ausreicht und der WR ab und zu neustartet.

Warum macht ihr so viele anfragen ? ich frage nur alle 10 sekunden ab, macht bei 2 Wechselrichtern alle 20 Sekunden ist einer dran. 
Eigentlich dürfen sowieso nicht so viele Anfragen sein im dem 868MHz bereich gilt ja nach frequenz ein Dutycycyle vom 1% oder 10% (senden pro Stunde).

**Reply by tah78** [reply]
Bei mir läuft es seit Änderung der Frequenz mal wieder einige Tage durchgehend. Auffällig ist nur der sehr wechselhafte RSSI, insb. bei dem WR, der sonst immer Probleme hat. 

Mein Intervall ist so klein weil ich es für PV gestütztes Laden nutze. Es muss theoretisch nicht so minimal sein, da auch die Steuerung der Ladeelektronik etwas träge ist. Aber ich hatte damit auch schon experimentiert und keinen Unterschied festgestellt. Deshalb verwende ich das Minimum, was bei mir bedeutet, dass jeder WR alle 9 Sekunden angefragt wird.

@turrican944, was bedeutet das in der Praxis:
>Eigentlich dürfen sowieso nicht so viele Anfragen sein im dem 868MHz bereich gilt ja nach frequenz ein Dutycycyle vom 1% oder 10% (senden pro Stunde).

**Reply by turrican944** [reply]
Das bedeutet das die DTU und die Wechselrichter je nach eingestellter Frequenz max. 1% (36 sek) bzw 10% (360sek) in einer Stunde senden dürfen. 
Wie lange so ein Datenpaket dauert weis ich grade nicht. Ob die originale DTU das einhält keine Ahnung. 
Ich bin der Meinung des es mit den 10 Sek und zwei WR gerade so gepasst hat. Ich wohne hier auf dem Land wo das wohl keiner merk, wenn du aber Pech hast und der Störmessdienst kommt und das feststellt musst du abschalten. Es gab irgendwie mal einen Artikel zur original DTU das die das wohl auch nicht einhält.

**Comment by ms49434** [comment]
Hier noch einmal die Liste mit den Frequenzen, die sich in der Hoymiles DTU App auswählen lassen:

1. 863.00 MHz
2. 863.75 MHz
3. 864.50 MHz
4. 865.25 MHz
5. 866.00 MHz
6. 866.75 MHz
7. 867.50 MHz
8. 868.25 MHz
9. 869.00 MHz
10. 869.75 MHz

Die Werte wurden mit einem tiny Spectrum Analyzer ermittelt.
